File: Backend/BM-F1-WorkshopInventory/BM-F1-WorkshopInventory/Models/Entities/GrandPrixResult.cs
namespace BM_F1_WorkshopInventory.Models.Entities
{
    public class GrandPrixResult
    {
        public Guid Id { get; set; }

        public DateOnly RaceDay {get; set ;}

        public string Location { get; set; } = "Silverstone";

        public double PointsScored { get; set; }
        
        public int PositionInTeamGrid { get; set; }

        public User CreatedBy { get; set; } = null!;
    }
}

File: Frontend/BM-F1-Frontend/src/Interfaces/GrandPrixResult.ts
export interface GrandPrixResult{
    raceDay: string,
    location: string,
    pointsScored: number,
    positionInTeamGrid: number
}

export interface DbTracked{
    id:string,
}

export interface DbTrackedGrandPrixResult extends DbTracked,GrandPrixResult{}